pelican-bootstrap3/templates/includes/sidebar-images.html

```diff
@@ -7,6 +7,9 @@
         {% for image in SIDEBAR_IMAGES %}
             <img width="100%" class="img-thumbnail" src="{{ image }}"/>
         {% endfor %}
+        {% for name, attributes in SIDEBAR_IMAGES_WITH_ATTRIBUTES.iteritems() %}
```

Comment:
How about storing the data in an array instead of a list?  Array would allow for controlling order.